subsystems/sidecar/index.js

```diff
@@ -697,7 +699,7 @@ class Sidecar extends ReadyResource {
     const id = client.userData?.id || `${client.id}@${startId}`
     const app = client.userData = client.userData?.id ? client.userData : new this.App({ id, startId, session })
     await this.ready()
-    const dht = this.swarm.dht.toArray({ limit: 20 })
+    const dht = { nodes: this.swarm.dht.toArray({ limit: KNOWN_NODES_LIMIT }) || [] }
```

Comment:
I think it should be: `const dht = { nodes: this.swarm.dht.toArray({ limit: KNOWN_NODES_LIMIT }) || [], bootstrap: this.dhtBootstrap }`

Reply:
in the same spirit, very important!, if custom bootstraps are set DO NOT persist or load the node cache from disk. will create intense debugging scenarios